### PR TITLE
Fix scaling issues

### PR DESCRIPTION
**Pull request by mdecleir**
I fixed several issues in the LXD scaling:
- Make sure the corrected response curve is used for the integrations.
- Update the wavelengths of the IRAC response curves to Angstrom (as assumed by the integration method).
- Make sure you are only integrating over the wavelength range of the bandpass (0 transmission outside of the band).
- Take into account gaps in the spectrum, due to bad regions. These regions should be bridged (interpolated), and not have flux=0.

### CODE REVIEW

File: measure_extinction/utils/plot_bandpasses.py [hunk 1]
```diff
@@ -28,5 +28,5 @@ def plot_bandpasses(bands):
     band_path = pkg_resources.resource_filename("measure_extinction", "data/Band_RespCurves/")
 
     # define the different bandpasses
-    bands = ["JohnU", "JohnB", "JohnV", "JohnR", "JohnI", "JohnJ", "2MASSJ", "JohnH", "2MASSH", "JohnK", "JohnKs", "2MASSKs", "AAOL", "IRAC1", "AAOLprime", "IRAC2", "AAOM", "IRAC3", "IRAC4"]
+    bands = ["JohnU", "JohnB", "JohnV", "JohnR", "JohnI", "JohnJ", "2MASSJ", "JohnH", "2MASSH", "JohnK", "JohnKs", "2MASSKs", "AAOL", "IRAC1", "AAOLprime", "IRAC2", "AAOM"]
```
Comment (karllark) [inline measure_extinction/utils/plot_bandpasses.py:31]: What happened to IRAC3 and IRAC4?

Comment (karllark) [reply measure_extinction/utils/plot_bandpasses.py:31]: Not used for SpeX?

Comment (mdecleir) [reply measure_extinction/utils/plot_bandpasses.py:31]: They are at larger wavelengths and the plot becomes unclear if you include so many bands, so I limited to those relevant for SpeX (and lower). But you can add/remove any band that you need basically.

Comment (karllark) [reply measure_extinction/utils/plot_bandpasses.py:31]: Cool.  Thanks for the info.

**Pull request merged**